draft-schinazi-masque-h3-datagram.md

```diff
@@ -165,51 +165,71 @@ with error H3_SETTINGS_ERROR.
 # Datagram-Flow-Id Header Field Definition {#header}
 
 "Datagram-Flow-Id" is a Item Structured
-Field {{!STRUCT-FIELD=I-D.ietf-httpbis-header-structure}}. Its value MUST be an
-Integer. Its ABNF is:
+Field {{!STRUCT-FIELD=I-D.ietf-httpbis-header-structure}}. Its value MUST be a
+List Structured Field, whose members MUST all be Items of type Integer. Its
```

Comment:
I think its simpler to write just

"Datagram-Flow-Id" is a List Structured Field {{!STRUCT-FIELD=I-D.ietf-httpbis-header-structure}}, whose members MUST all be Items of type Integer. Its ABNF is:"

(GitHub prevents me from making this as an actual direct suggestion)

Reply:
Oh I should have read my work more carefully, you're absolutely right. Done.